src/state/selection/reducer.ts

```diff
@@ -28,8 +28,8 @@ import {
 export const initialState = {
     time: 0,
     numberPanelsCollapsed: 0,
-    visibleAgentKeys: {},
-    highlightedAgentKeys: {},
+    agentVisibilityMap: {},
+    agentHighlightMap: {},
     agentColors: {},
```

Comment:
based on the basic selectors for `selection` it seems like initalState is either missing some keys, or there are some old selectors that can be deleted. like `export const getAgentsOnById = (state: State) => state.selection.agentIdsOn;`

Reply:
Good catch, looks like `getAgentsOnById` is not used anywhere. I just checked all the basic selectors (had only checked composed ones) and removed the unused ones.